linkfiles/.golangci.yaml

```
@@ -1,4 +1,4 @@
-# Options for analysis running.
+version: "2"
 run:
   # Timeout for analysis, e.g. 30s, 5m.
   timeout: 5m
```

I recently found out about this configuration, which allows you to run more than one golangci-lint instance at a time on your machine. I suggest we include it as it doesn't seem to have any negative impacts in my testing, and I have some parallelization stuff on my radar that will require this at some point in the future.

```suggestion
  timeout: 5m
  allow-parallel-runners: true
```